selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/KeepDirectoryGitImportTest.java

```diff
@@ -149,10 +154,14 @@ public void keepDirectoryImportFromGitHub() throws Exception {
     loader.waitOnClosed();
     importProject.waitLoadRepoBtn();
     importProject.clickLoadRepoBtn();
+
+    authorizeGitHubApplication();
```

Comment:
Do we really need `authorizeGitHubApplication` if we stop to connect to test repo by git ssh url?